editor/components/inserter/menu.js

```diff
@@ -176,6 +182,14 @@ export class InserterMenu extends Component {
 				/>
 
 				<div className="editor-inserter__results">
+					<ChildBlocks
+						rootBlockIcon={ rootBlockIcon }
```

Comment:
Did you consider to use `withSelect` directly on `ChildBlocks` component instead of passing down `rootBlockIcon` and `rootBlockTitle`?